src/Codec/Xlsx/Types/StyleSheet.hs

```diff
@@ -1176,6 +1232,23 @@ instance ToElement CellXf where
         ]
     }
 
+-- | See @CT_CellStyle@, p. 3937
+instance ToElement CellStyle where
+  toElement nm CellStyle{..} = Element {
+      elementName       = nm
+    , elementNodes      = map NodeElement . catMaybes $ [
+          -- TODO: extLst
```

Comment:
no need for such a TODO (but you could put it into `CellStyle` definition if you'd like to).

Also this could be simplified a bit using `leafElement` from `Codec.Xlsx.Writer.Internal`